### ISSUE TITLE
No way to recover cluster

### ISSUE THREAD

**Issue by seanfulton** [distribworks/dkron#999]
I had a working, 3-node dkron cluster-A

I created a second, 3-node dkron cluster-B

I accidentally restarted the nodes with cluster-A;s IP addresses in them. I stopped cluster-B, changed the IP addresses, and restarted cluster-B. It still saw the first cluster and the leader (of now a six-node cluster) was in cluster-A.

I stopped all nodes in cluster-A and cluster-B. I deleted the contents of /dkron.data on all six nodes.

I restarted cluster-A and cluster-B. Cluster-A works as expected; dkron raft shows three nodes

Cluster B shows all three nodes as followers and refuses to select a leader.

I have tried both with and without bootstrap-expect:
# Dkron example configuration file

# This node is running in server mode
server: true

# Provides the number of expected servers in the datacenter.
# Either this value should not be provided or the value must agree with other servers in the cluster.
# When provided, Dkron waits until the specified number of servers are available and then bootstraps the cluster.
# This allows an initial leader to be elected automatically. This flag requires server mode.
#bootstrap-expect: 3

bind-addr: "{{ GetPrivateIP }}:8946"

join:
- 10.10.20.142
- 10.10.20.143
- 10.10.20.144

log-level: debug
http-addr: ":8090"

**Comment by seanfulton** [comment]
This is absolutely infuriating. I have deleted all data folders, reinstalled the software (different version) and followed the instructions for boot-strapping a new cluster. I can't get this to start.

**Comment by yvanoers** [comment]
Can you provide logs, the version of Dkron and how your platform is configured (docker, VMs, OS, etc)?
Ideally I'd like to log in on your cluster and have a look myself, but that is probably out of the question :)

**Comment by victorcoder** [comment]
Yes, please, startup log for cluster B would be useful, also doublecheck you configuration IPs, it's easy to bootstrap a cluster with clean data.

Also, does cluster B works bootstrapping with a single node?

**Comment by vcastellm** [comment]
Closing for lack of info
